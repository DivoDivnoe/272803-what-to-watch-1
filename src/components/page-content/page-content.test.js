import React from 'react';
import renderer from 'react-test-renderer';
import PageContent from './page-content.jsx';

const MockComponent = () => <div />;

describe(`PageContent component`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
        <PageContent>
          <MockComponent />
        </PageContent>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
