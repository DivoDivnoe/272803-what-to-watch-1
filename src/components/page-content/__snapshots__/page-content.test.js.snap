// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent component renders correctly 1`] = `
<div
  className="page-content"
>
  <div />
</div>
`;
